projects/templates/plopfile.js

```diff
@@ -40,22 +36,43 @@ module.exports = function (plop) {
     });
 
     plop.setActionType('install deps', function (answers) {
-        execSync(
-            `cd ../../ && yarn lerna add @spectrum-web-components/base --scope=@spectrum-web-components/${answers.name} --no-bootstrap`
-        );
-        execSync(
-            `cd ../../ && yarn lerna add @spectrum-web-components/${answers.name} --scope=@spectrum-web-components/bundle --no-bootstrap`
-        );
-        if (answers.spectrum)
+        try {
+            // Add base as a workspace dependency to the new package
+            execSync(
+                `cd ../../ && yarn workspace @spectrum-web-components/${answers.name} add @spectrum-web-components/base@workspace:^`
+            );
+            // Add the new package to bundle with a fixed version
             execSync(
-                `cd ../../ && yarn lerna add @spectrum-css/${answers.spectrum} --scope=@spectrum-web-components/${answers.name} --dev --no-bootstrap`
+                `cd ../../ && yarn workspace @spectrum-web-components/bundle add @spectrum-web-components/${answers.name}@0.0.1`
             );
+        } catch (error) {
+            // Silently fail, dependencies will need to be added manually
+        }
     });
 
     plop.setActionType('format files', function (answers) {
-        execSync(
-            `cd ../../ && yarn prettier --write packages/${answers.name} && yarn eslint --fix -f pretty packages/${answers.name} && yarn stylelint --fix packages/${answers.name}`
-        );
+        try {
```

Comment:
Seems like we can skip this part in the templating because it would get linted before we commit anyway.